### DISCUSSION TITLE
(first time only) make the next button in setup wizard have atleast 5 second cooldown

### DISCUSSION THREAD

**Discussion by anispwyn** [ppy/osu#18278] [Ideas] [answered]
this is kinda useless feature but

to avoid people skipping the setup wizard this is great things to make people atleast read what is happening on screen

**Comment by bdach** [comment, marked as answer]
You can quit out of the setup wizard at any time. If anything forcing people to read will frustrate them rather than make them go through it.

Let's have some trust that users know what they're doing rather than try and railroad them through the process, IMO.
